README.md

```diff
@@ -1,3 +1,32 @@
-# Datadog CloudFoundry Buildpack
+# Datadog Cloud Foundry Buildpack
 
-This buildpack installs Dogstatsd into the container your app is running on.
+This is a [decorator buildpack](https://github.com/cf-platform-eng/meta-buildpack/blob/master/README.md#decorators) for Cloud Foundry. It will install a Datadog Dogstatsd binary on the container your app is running on. It uses the new Agent 6 build of Dogstatsd, which is very light weight and powerful. You should barely notice it running on your container.
+
+## Use
+
+### Install the [Meta Buildpack](https://github.com/cf-platform-eng/meta-buildpack#how-to-install-the-meta-buildpack)
+First, you will have to [install the Meta Buildpack](https://github.com/cf-platform-eng/meta-buildpack#how-to-install-the-meta-buildpack). This enables apps to use decorator buildpacks. Follow the instructions to get the buildpack and upload it if you don't already have it.
+
+### Upload the Datadog Cloud Foundry Buildpack
+You can get it from the [releases page](https://github.com/DataDog/datadog-cloudfoundry-buildpack/releases) here, or you can just [download the latest version directly from us](https://cloudfoundry.datadoghq.com/datadog-cloudfoundry-buildpack/datadog-cloudfoundry-buildpack-latest.zip). After you download the zipfile, you will have to upload it to Cloud Foundry:
```

Comment:
Do we need to offer both links? Should we just point at Datadog?

Download the latest Datadog [build pack release]((https://cloudfoundry.datadoghq.com/datadog-cloudfoundry-buildpack/datadog-cloudfoundry-buildpack-latest.zip). After you downloading the zipfile, you will have to upload it to your Cloud Foundry environment.